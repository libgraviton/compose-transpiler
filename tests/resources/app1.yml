components:
  fred:
    expose:
      doExpose: true
    mixins:
      more:
  hans:
    mixins:
      more:
        derDude: yes
