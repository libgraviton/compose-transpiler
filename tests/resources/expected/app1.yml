version: 99
services:
  fred:
    image: 'fred:latest'
    addedKey: yeah
  fred-expose:
    image: 'nginx:5.5.5'
  hans:
    image: 'hans:2.0.0'
    addedKey: yeah
    oneMoreKey: why
  my-setup:
    image: 'org/xckb-setup:3.0.0'
networks:
  internal:
    external: false
